src/main/java/core/screens/GameScreen.java

```diff
@@ -21,7 +20,16 @@
     private float x;
     private float y;
 
-    public GameScreen() {}
+    private final HudManager hud;
+    private final GdxGame game;
+    private boolean displayInventory = false;
+    private InventoryWindow inventory;
```

Comment:
move to HUDManager

inventoryWindow is not the only thing that's gonna show up on the HUD, all GameplayController should care about is having an HUD

Reply:
That's a really good point! I moved the InvWindow and display boolean to HUDManager.